web3/manager.py

```diff
@@ -150,6 +179,9 @@ def request_blocking(
 
         if "error" in response:
             apply_error_formatters(error_formatters, response)
```

Comment:
Does this line have any effect? (This comment is unrelated to the changes in this PR)